src/libmongoc/src/mongoc/mcd-azure.c

```diff
@@ -132,7 +133,7 @@ mcd_azure_access_token_try_init_from_json_str(mcd_azure_access_token *out,
                            mlib_in_range(int, expires_in_len) ? (int)expires_in_len : INT_MAX,
                            expires_in_str);
       } else {
-         out->expires_in = mcd_seconds(s);
+         out->expires_in = mlib_duration(s, s);
```

Comment:
```suggestion
         out->expires_in = mlib_duration(expires_in, s);
```
Avoid confusing reuse of `s`.
```c
const long long expires_in = strtoll(expires_in_str, &parse_end, 0);
```